circleci/templates/maven/config_template-no-docker-v1.yml

```diff
@@ -109,13 +116,47 @@ jobs:
       - image: circleci/ruby:2.4.1
     steps:
       - run: echo "*PLACEHOLDER* pretending to execute integration testing phase!"
-  Deploy Artifacts: # Deploy artifacts to Nexus or Maven Central
-    docker:
-      - image: circleci/ruby:2.4.1
-    steps:
-      - run: echo "*PLACEHOLDER* pretending to deploy build artifacts to Maven Central!"
 # END: UNDER CONSTRUCTION AREA ---------------------------------------
 
+  Tag Release:
+    docker: *image
+    steps:
+      # Tag Release creates a Git tag with given a given name and pushes it
+      # This job is governed by the following environment variables:
+      # $RELEASE - Set to true to perform a release, otherwise this step is skipped
+      # $RELEASE_TAG - The release tag for this version
+      # $RELEASE_VERSION - The version to release
+      # $NEXT_VERSION - The next development version
+      # To use this step the following MUST also be set: $GITHUB_KEY and $ION_GPG_KEYRING
+      - checkout
+      - run:
+          name: Tag Release
+          command: |
```

Comment:
Let's add a comment here that we are modifying a file inside a container and pushing it back to the github repo?

Reply:
All set